server/storage/schema/cindex.go

```diff
@@ -70,6 +70,17 @@ func UnsafeUpdateConsistentIndex(tx backend.BatchTx, index uint64, term uint64,
 		if term == oldTerm && index <= oldi {
```

Comment:
At this point I think it would be more readable without this line and instead

```
if (index > oldi) {
  bs1 := make([]byte, 8)
  binary.BigEndian.PutUint64(bs1, index)
  // put the index into the underlying backend
  // tx has been locked in TxnBegin, so there is no need to lock it again
  tx.UnsafePut(Meta, MetaConsistentIndexKeyName, bs1)
}
```